src/server_manager/web_app/mocks/mocks.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The Outline Authors
```

Comment:
This is app-agnostic code. You can move to `infrastructure/`
Also, `mock` isn't a great name. This is a valid Storage, implemented in memory.
Maybe name it `memory_storage.ts`?

Reply:
Done. I also made the prompt impression delay configurable to have faster tests.